TKGame/Entity.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace TKGame
 {
-    abstract class Entity
+    public abstract class Entity
```

Comment:
Is this temporary or will it need to be public henceforth?

Reply:
I'm guessing temporary. I changed to public because of access issues with DrawBoundingRectangle()